HACKING.md

```diff
@@ -57,6 +57,20 @@ What do I need to do to test...
 
 1. Build and push a custom release image to your own repository.
 
+         docker build --platform linux/amd64 -t quay.io/blah/controlplaneoperator:<tag> -f Dockerfile.control-plane .
```

Comment:
I would suggest to use the dev Dockerfile here to build both the HO and CPO image at the same time - https://github.com/openshift/hypershift/blob/fdd76ee5a56a9384216e1fa87ab5919f4a59fa19/Dockerfile.dev#L1